### ISSUE TITLE
Paging not working for admin Groups UI

### ISSUE THREAD

**Issue by dbarentine** [keycloak/keycloak-ui#3558]

**Comment by edewit** [comment]
We recently changed how groups work the tree view and table are now combined into one:
![image](https://user-images.githubusercontent.com/51133/195349046-f56da924-d339-4e1d-b82a-8ff16f1f0a75.png)
I think it will work better for you this way, you can check it out by using the [nightly version](https://github.com/keycloak/keycloak/releases/tag/nightly)

**Comment by ssilvert** [comment]
I believe this is a duplicate of https://github.com/keycloak/keycloak-ui/issues/3517

**Comment by dbarentine** [comment]
@ssilvert yes I missed that one in my search but it does look like a duplicate. Although I will add that I believe the search functionality is broken currently as well. That may be related to the paging problem though. I'm fine if you want close this one in favor of #3517 

@edewit I'll take a look at the nightly when I get a chance. I'm assuming that is targeted for the 20.0.0 release.

**Comment by edewit** [comment]
yes

**Comment by yaoguoh** [comment]
Docker image ’quay.io/keycloak/keycloak:19.0.3-legacy‘ deployed.
Nightly changes a little big, It is not very convenient to use without ‘/auth’ routing.
